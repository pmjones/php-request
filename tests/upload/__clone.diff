001+ fail
001- ok