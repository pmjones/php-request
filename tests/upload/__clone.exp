ok
string(3) "foo"