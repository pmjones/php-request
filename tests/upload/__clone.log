
---- EXPECTED OUTPUT
ok
string(3) "foo"
---- ACTUAL OUTPUT
fail
string(3) "foo"
---- FAILED
