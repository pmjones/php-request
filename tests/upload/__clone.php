<?php

$upload = new SapiUpload('foo', 'text/plain', 123, '/tmp/foo.txt', 0);

$clone = clone $upload;
try {
    $clone->name = 'bar';
    echo "fail\n";
} catch( Exception $e ) {
    echo "ok\n";
}
var_dump($upload->name);

