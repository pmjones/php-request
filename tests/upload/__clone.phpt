--TEST--
SapiUpload::__clone
--FILE--
<?php

$upload = new SapiUpload('foo', 'text/plain', 123, '/tmp/foo.txt', 0);

$clone = clone $upload;
try {
    $clone->name = 'bar';
    echo "fail\n";
} catch( Exception $e ) {
    echo "ok\n";
}
var_dump($upload->name);

--EXPECT--
ok
string(3) "foo"
