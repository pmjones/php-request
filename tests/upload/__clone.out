fail
string(3) "foo"